app/js/daemonManager.js

```diff
@@ -73,7 +73,7 @@ function DaemonManager() {
 			if (err) {
 				self.Running = false;
 				// Check again later
-				setTimeout(updatePrompt, 1000);
+				setTimeout(updatePrompt, 100000); // Not great - we'll have to fix that.
```

Comment:
goal was to wait until the wallet had finished unlocking, according to a comment. 1 second is definitely not enough even on my desktop.

but maybe 100 seconds is too many. I'm not even sure it does anything.